models/evc.py

```diff
@@ -1244,13 +1309,22 @@ def _install_uni_flows(
     ) -> dict[str, list[dict]]:
         """Install UNI flows."""
         uni_flows = self._prepare_uni_flows(path, skip_in, skip_out)
-
-        for (dpid, flows) in uni_flows.items():
-            self._send_flow_mods(dpid, flows)
+        try:
+            for (dpid, flows) in uni_flows.items():
+                self._send_flow_mods(dpid, flows)
+        except FlowModException as err:
+            raise EVCPathNotInstalled(f"In {dpid}, {err}")
 
         return uni_flows
 
     @staticmethod
+    @retry(
+        stop=stop_after_attempt(3),
+        wait=wait_combine(wait_fixed(3), wait_random(min=2, max=7)),
+        retry=retry_if_exception_type(FlowModException),
```

Comment:
timeouts aren't still handled

Reply:
Added `timeout=10`

Reply:
Explored stress test with default value which is 5 seconds, it should be fine with 10. Changed to 30 nonetheless.